include/fmt/chrono.h

```diff
@@ -2052,16 +2052,17 @@ struct formatter<std::chrono::duration<Rep, Period>, Char> {
   }
 };
 
-template <typename Char, typename Duration>
-struct formatter<std::chrono::time_point<std::chrono::system_clock, Duration>,
+template <typename Rep, typename Period, typename Char>
```

Comment:
Why change template parameters? I suggest keeping the old ones at least for consistency with utc formatter and change the `format` function instead.

Reply:
We need ``Rep`` and ``Period`` types for the condition on lines 2070-2071.

Reply:
Yeah but these are available as `Duration::rep` and `Duration::period`. I don't think we need them here.

Reply:
Done